{
  "classifiers": [
    "development status :: 6 - mature",
    "framework :: pytest",
    "intended audience :: developers",
    "license :: osi approved :: mit license",
    "operating system :: os independent",
    "topic :: software development :: testing"
  ],
  "description": "===================\npytest-faulthandler\n===================\n\n\nplugin for pytest that automatically enables the\n`faulthandler <https://faulthandler.readthedocs.io/>`_ module during tests.\n\ninspired by the\n`nose faulthandler <https://github.com/schlamar/nose-faulthandler>`_ plugin.\n\n**this plugin is now part of pytest core since pytest 5.0, so users should not install this plugin together with that pytest version.**\n\n**version 2.0.0 of this package is a dummy package for pytest 5.0 compatibility**.\n\n\nusage\n=====\n\nthe plugin is always active by default, but you can disable it by passing\n``--no-faulthandler`` to ``py.test``.\n\noptions:\n\n* ``--faulthandler-timeout=timeout``: dump the traceback of all threads if a\n  test takes more than timeout seconds to finish.\n\n\nrequirements\n============\n\n* python 2.7+, python 3.4+\n* pytest\n* faulthandler (python 2.7)\n\n\ninstall\n=======\n\ninstall using `pip <http://pip-installer.org/>`_:\n\n.. code-block:: console\n\n    $ pip install pytest-faulthandler\n\nchange log\n==========\n\nplease consult the `changelog`_.\n\n.. _changelog: https://github.com/pytest-dev/pytest-faulthandler/blob/master/changelog.rst\n\n\n",
  "docs_url": null,
  "keywords": "pytest faulthandler",
  "license": "mit",
  "name": "pytest-faulthandler",
  "package_url": "https://pypi.org/project/pytest-faulthandler/",
  "project_url": "https://pypi.org/project/pytest-faulthandler/",
  "project_urls": {
    "Homepage": "https://github.com/pytest-dev/pytest-faulthandler"
  },
  "release_url": "https://pypi.org/project/pytest-faulthandler/2.0.1/",
  "requires_dist": [
    "pytest (>=5.0)"
  ],
  "requires_python": "",
  "summary": "py.test plugin that activates the fault handler module for tests (dummy package)",
  "version": "2.0.1",
  "releases": [],
  "developers": [
    "bruno_oliveira",
    "nicoddemus@gmail.com"
  ],
  "kwds": "pytest faulthandler fault py tests",
  "license_kwds": "mit",
  "libtype": "pypi",
  "id": "pypi_pytest_faulthandler",
  "homepage": "https://github.com/pytest-dev/pytest-faulthandler",
  "release_count": 14,
  "dependency_ids": [
    "pypi_pytest"
  ]
}